py3status/modules/fedora_updates.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Displays the number of package updates pending for a Fedora Linux installation.
+This will display a count of how many `dnf` updates are waiting
```

Comment:
please add a blank line above this line (separates the short description with the long one)